doc/null_safety/null_safe_migration.md

```diff
@@ -16,29 +17,76 @@ If you haven't already, familiarize yourself with the concepts of Dart null safe
 - Dart's [Understanding null safety](https://dart.dev/null-safety/understanding-null-safety) article 
 - OverReact's documentation on [null safety and required props](../null_safety_and_required_props.md)
 
-## Step 2: Run the preparation codemod (coming soon!)
+## Step 2: Run the preparation codemods
 
-<!-- TODO - update this section once we release the `null_safety_prep` codemod. -->
-
-> [!WARNING]
-> This codemod does not exist yet, but is coming soon!
-
-Start out by running the `null_safety_prep` [codemod][orcm] on your repo:
+Start out by running the [codemod][orcm]s on your repo:
 
 ```
 dart pub global activate over_react_codemod
-dart pub global run over_react_codemod:null_safety_prep
+dart pub global run over_react_codemod:null_safety_prep --yes-to-all
```

Comment:
I think should probably recommend doing this one as a separate PR, before the migration, so that all those changes can be reviewed and merged separately from the larger null safety hint and migration commits.

Reply:
https://github.com/Workiva/over_react/pull/932/commits/911f5c9655578a8760d78a17113fb7e22d370a9c